src/main/java/mate/academy/exception/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message, Throwable e) {
+        super(message, e);
+    }

```

Comment:
what about case, when we use exception without user message? Try to overload it)